docs/source/algorithms/LoadISISNexus-v2.rst

```diff
@@ -14,13 +14,118 @@ Loads a Nexus file created from an ISIS instrument.
++----------------+-------------------------------------------------+
+| ``start_time`` | ``run_start``                                   |
++----------------+-------------------------------------------------+
+| ``end_time``   | ``run_end``                                     |
++----------------+-------------------------------------------------+
+| (data)         | ``nspectra``                                    |
++----------------+-------------------------------------------------+
+| (data)         | ``nchannels``                                   |
++----------------+-------------------------------------------------+
+| (data)         | ``nperiods``                                    |
++----------------+-------------------------------------------------+
+| ``IRPB[0]``    | ``dur``                                         |
++----------------+-------------------------------------------------+
+| ``IRPB[1]``    | ``durunits``                                    |
++----------------+-------------------------------------------------+
+| ``IRPB[2]``    | ``dur_freq``                                    |
++----------------+-------------------------------------------------+
+| ``IRPB[3]``    | ``dmp``                                         |
++----------------+-------------------------------------------------+
+| ``IRPB[4]``    | ``dmp_units``                                   |
++----------------+-------------------------------------------------+
+| ``IRPB[5]``    | ``dmp_freq``                                    |
++----------------+-------------------------------------------------+
+| ``IRPB[6]``    | ``freq``                                        |
++----------------+-------------------------------------------------+
+| ``IRPB[7]``    | ``gd_prtn_chrg``                                |
++----------------+-------------------------------------------------+
+| ``RRPB[9]``    | ``goodfrm``                                     |
++----------------+-------------------------------------------------+
+| ``RRPB[10]``   | ``rawfrm``                                      |
++----------------+-------------------------------------------------+
+| ``RRPB[11]``   | ``dur_wanted``                                  |
++----------------+-------------------------------------------------+
+| ``RRPB[12]``   | ``dur_secs``                                    |
++----------------+-------------------------------------------------+
+| ``RRPB[13]``   | ``mon_sum1``                                    |
++----------------+-------------------------------------------------+
+| ``RRPB[14]``   | ``mon_sum2``                                    |
++----------------+-------------------------------------------------+
+| ``RRPB[15]``   | ``mon_sum3``                                    |
++----------------+-------------------------------------------------+
+| ``RRPB[21]``   | ``rb_proposal``                                 |
```

Comment:
The algorithm does not read everything in the Nexus file. Part of the purpose of this issue is to document what it does read.